.gitlab-ci.yml

```diff
@@ -535,7 +576,8 @@ scenario_tests_linux_python35:
   script:
     - apk add bash
     - cd scenario-tests
-    - ./run_scenario_tests.sh --docker-python3.5 ../target/turicreate-*.whl
+    - ./run_scenario_tests.sh --docker-python3.5 \
+      "../target/$(ls ../target | grep turicreate | grep -v minimal)"
```

Comment:
Could just do this with a * right?  `"../target/turicreate*minimal*"`

Reply:
That's right!

Reply:
done.